### DISCUSSION TITLE
umi配置里面的proxy可以配置自定义请求头么，如果是基于http-proxy-middleware开发的应该是可以的，但是试了一下没有生效

### DISCUSSION THREAD

**Discussion by liujiajun1994** [umijs/umi#11253] [Q&A]
https://juejin.cn/s/http-proxy-middleware%20request%20headers
类似这样配置

**Comment by fz6m** [comment]
看一下这个 https://github.com/umijs/umi/issues/10760#issuecomment-1471158059 ，可以自定义额外的请求头
